src/cuga/backend/server/main.py

```
@@ -1629,6 +1629,95 @@ async def event_stream(
 app.include_router(secrets_routes.router)
 
 
+if getattr(settings, "a2a", None) and getattr(settings.a2a, "enabled", False):
+    # The A2A package is only imported when explicitly enabled in settings,
+    # so disabled deployments pay no import-time cost for it.
+    from cuga.backend.server.a2a import build_router as _build_a2a_router  # noqa: E402
+
+    class _A2AStreamEvent:
+        """Duck-typed event the A2A task adapter consumes."""
+
+        __slots__ = ("name", "data", "final")
```

Ruff (0.15.15)

[warning] `_A2AStreamEvent.__slots__` is not sorted

Apply a natural sort to `_A2AStreamEvent.__slots__`

(RUF023)